docs/agent/changes.md

```diff
@@ -1,6 +1,13 @@
-# Agent v6 Changes
+# Agent v7 vs v6
 
 This information is available in the Datadog documentation:<br>
-[docs.datadoghq.com/agent/faq/agent-v6-changes][1]
+[docs.datadoghq.com/agent/agent/versions?tab=agentv7vsv6][1]
```

Comment:
No need to hard-code the tab here, since this link will default to the latest version which is agent v7
```suggestion
[docs.datadoghq.com/agent/agent/versions][1]
```